doc/sphinx/language/core/definitions.rst

```diff
@@ -56,13 +56,13 @@ has type :n:`@type`.
 Top-level definitions
 ---------------------
 
-Definitions extend the global environment with associations of names to terms.
+Definitions extend the global environment by associating names to terms.
 A definition can be seen as a way to give a meaning to a name or as a
 way to abbreviate a term. In any case, the name can later be replaced at
 any time by its definition.
 
 The operation of unfolding a name into its definition is called
-:math:`\delta`-conversion (see Section :ref:`delta-reduction`). A
+:math:`\delta`-conversion (see Section :ref:`delta-reduction-sect`). A
```

Comment:
Maybe you can use the defined glossary term `delta-reduction` (instead of `delta-conversion`) to avoid having to link to the section and to allow removing this specific anchor.